resources/migrations/5_v0.5.14.down.sql

```diff
@@ -0,0 +1,148 @@
+--aws_apigatewayv2_vpc_links
+ALTER TABLE "aws_apigatewayv2_vpc_links"
+    ADD COLUMN "vpc_link_id" TEXT;
+
+UPDATE "aws_apigatewayv2_vpc_links"
+SET "vpc_link_id" = "id";
```

Comment:
the down would be to rename id -> vpc_link_id, I think you need to update the constraint double check me on that.